### PR TITLE
Strimzi Kafka was implemented

### LINKED ISSUE #3: Implement Kafka Strimzi

### PR DESCRIPTION
**Pull request by vpetrovaa**

### CODE REVIEW

File: infra/cars-secrets.yml [hunk 1]
```diff
@@ -4,4 +4,4 @@ metadata:
   name: cars-secrets
 type: Opaque
 data:
-  KAFKA_PORT: a2Fma2Etc2VydmljZTo5MDky
\ No newline at end of file
+  KAFKA_PORT: bXktY2x1c3Rlci1rYWZrYS1ib290c3RyYXA6OTA5Mg==
```
Comment (h1alexbel) [inline infra/cars-secrets.yml]: @vpetrovaa why do we need to store `port` in the Secret?

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: infra/cars-secrets.yml (hunk 1 above)
Comment (vpetrovaa) [reply infra/cars-secrets.yml]: @h1alexbel this secret means host+port. Change to only host?

Comment (h1alexbel) [reply infra/cars-secrets.yml]: @vpetrovaa I think we can move this information into deployment, host can be stored in the `ConfigMap`

Comment (vpetrovaa) [reply infra/cars-secrets.yml]: @h1alexbel fixed https://github.com/vpetrovaa/carsharing-k8s/pull/4/commits/ee75e3ba328a5af690d26cc13612ad87ce88443d

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
lgtm

**Pull request merged**